pkg/sensors/test/checker.go

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package test
+
+import (
+	"os/exec"
+	"runtime"
+	"testing"
+
+	"github.com/cilium/tetragon/api/v1/tetragon"
+	"github.com/cilium/tetragon/pkg/testutils"
+	"github.com/sirupsen/logrus"
+
+	ec "github.com/cilium/tetragon/api/v1/tetragon/codegen/eventchecker"
+)
+
+//revive:disable
```

Comment:
Why are we disabling revive here? Is it to do with the "stuttering" naming lint?

Reply:
yap!